### PR TITLE
v0.13.1

### PR DESCRIPTION
**Pull request by tmeyer2115**
# Version 0.13.1

## Changes

* Made all hard-coded strings in the various templates configurable.
* Updated the question submission endpoint hit by the SDK.
* Created a new UMD-style dist build of the SDK that includes polyfills.
* Added a LegacyCard component adapted ResultsItem.
* A new animation for the SearchBar component's icon.

## Fixes

* Added an aria-label to the SearchBar component's input field for WCAG compliance.
* Assorted small styling fixes.
* Ensured all facets, filters, and sorts are set correctly on page load.
* Improved backwards compatibility with previous versions by making minor alterations to the interface of some components.

### CODE REVIEW

File: src/ui/sass/modules/_Icon.scss [hunk 1]
```diff
@@ -11,16 +11,27 @@
     fill: currentColor;
   }
 
-  &-sm svg {
+  &--sm svg,
+  &--lg img
```
Comment (oshi97) [inline src/ui/sass/modules/_Icon.scss]: This --lg img should be --sm img

**Review by alexisgrow — DISMISSED** [review]
LGTM

**Review by oshi97 — APPROVED** [review]
lgtm!!

**Review by alexisgrow — APPROVED** [review]
LG!

**Pull request merged**